src/client.ts

```diff
@@ -42,6 +48,60 @@ class SpaceClient {
       );
     });
   }
+
+  txlSubscribe(
+    metadata: grpcWeb.Metadata = {},
+  ): grpcWeb.ClientReadableStream<TextileEventResponse> {
+    const request = new Empty();
+
+    return this.instance.txlSubscribe(request, metadata);
+  }
+
+  openFile(
+    path: string,
```

Comment:
should not be better to pass an object as payload? Thinking that the request payload could change and receive more than one field.

Reply:
@giankotarola  changed on b87d1b0